gym/lib/gym/error_handler.rb

```diff
@@ -54,6 +54,7 @@ def handle_build_error(output)
         print_xcode_path_instructions
         print_xcode_version
         print_full_log_path
+        print_environment_information
```

Comment:
I think we should always print those, otherwise we'll get more GH issues as people are not able to see why their build failed.